peerscout/keyword_extract/keyword_extract.py

```diff
@@ -313,14 +313,15 @@ def write_to_jsonl_file(
             write_file.write("\n")
 
 
-def iter_get_batches(iterable, size):
+def iter_get_batches(iterator: Iterator[T], size: int) -> Iterable[T]:
```

Comment:
renamed to `iterator` because it doesn't seem to accept iterables such as list directly